ui/src/importer/importerTaskDetails/ImporterTaskDetailsComponents/ImportedDocumentReport.js

```diff
@@ -221,25 +229,31 @@ class ImportedDocumentReportComponent extends Component {
         </Table.Cell>
         <Table.Cell collapsing>
           {!_isEmpty(documentReport.partial_matches) && (
-            <>
-              {' '}
-              <Icon color="red" name="exclamation" size="large" />
-              {documentReport.partial_matches.map(entry => {
-                return (
-                  <>
-                    {
-                      <Link
-                        to={BackOfficeRoutes.documentDetailsFor(entry.pid)}
-                        target="_blank"
-                      >
-                        {entry.pid}
-                      </Link>
-                    }{' '}
-                    <Label>{entry.type}</Label>
-                  </>
-                );
-              })}
-            </>
+            <Grid>
+              <Grid.Row>
+                <Grid.Column width={2}>
+                  <Icon color="red" name="exclamation" size="large" />
+                </Grid.Column>
+                <Grid.Column width={14}>
+                  <List>
```

Comment:
I think the list would be enough to push each element to a new row, adding the grid with one column is a bit overkill in my opinion. What do you think?